config_test.go

```diff
@@ -60,55 +60,114 @@ func TestClusterConfigToConfigString(t *testing.T) {
-	tests[len(tests)-1].clusterConfig.Consistency = 1
-	tests = append(tests, TestStringToConfigStruct{info: "Timeout < 0", configString: "?timeout=-1s", clusterConfig: NewClusterConfig()})
-	tests = append(tests, TestStringToConfigStruct{info: "Timeout > 0", configString: "?timeout=1s", clusterConfig: NewClusterConfig()})
-	tests[len(tests)-1].clusterConfig.Timeout = time.Second
-	tests = append(tests, TestStringToConfigStruct{info: "ConnectTimeout < 0", configString: "?connectTimeout=-1s", clusterConfig: NewClusterConfig()})
-	tests = append(tests, TestStringToConfigStruct{info: "ConnectTimeout > 0", configString: "?connectTimeout=1s", clusterConfig: NewClusterConfig()})
-	tests[len(tests)-1].clusterConfig.ConnectTimeout = time.Second
-	tests = append(tests, TestStringToConfigStruct{info: "Keyspace", configString: "?keyspace=system", clusterConfig: NewClusterConfig()})
-	tests[len(tests)-1].clusterConfig.Keyspace = "system"
-	tests = append(tests, TestStringToConfigStruct{info: "NumConns < 1", configString: "?numConns=0", clusterConfig: NewClusterConfig()})
-	tests = append(tests, TestStringToConfigStruct{info: "NumConns > 1", configString: "?numConns=2", clusterConfig: NewClusterConfig()})
-	tests[len(tests)-1].clusterConfig.NumConns = 2
-	tests = append(tests, TestStringToConfigStruct{info: "ignorePeerAddr true", configString: "?ignorePeerAddr=true", clusterConfig: NewClusterConfig()})
-	tests[len(tests)-1].clusterConfig.IgnorePeerAddr = true
-	tests = append(tests, TestStringToConfigStruct{info: "disableInitialHostLookup true", configString: "?disableInitialHostLookup=true", clusterConfig: NewClusterConfig()})
-	tests[len(tests)-1].clusterConfig.DisableInitialHostLookup = true
-	tests = append(tests, TestStringToConfigStruct{info: "writeCoalesceWaitTime 1s", configString: "?writeCoalesceWaitTime=1s", clusterConfig: NewClusterConfig()})
-	tests[len(tests)-1].clusterConfig.WriteCoalesceWaitTime = time.Second
-	tests = append(tests, TestStringToConfigStruct{info: "Host", configString: "one", clusterConfig: NewClusterConfig()})
-	tests[len(tests)-1].clusterConfig.Hosts = []string{"one"}
-	tests = append(tests, TestStringToConfigStruct{info: "Hosts", configString: "one,two,three", clusterConfig: NewClusterConfig()})
-	tests[len(tests)-1].clusterConfig.Hosts = []string{"one", "two", "three"}
-	tests = append(tests, TestStringToConfigStruct{info: "Host & Consistency any", configString: "one?consistency=any", clusterConfig: NewClusterConfig()})
-	tests[len(tests)-1].clusterConfig.Consistency = 0
-	tests[len(tests)-1].clusterConfig.Hosts = []string{"one"}
-	tests = append(tests, TestStringToConfigStruct{info: "Hosts & Consistency one", configString: "one,two,three?consistency=one", clusterConfig: NewClusterConfig()})
-	tests[len(tests)-1].clusterConfig.Consistency = 1
-	tests[len(tests)-1].clusterConfig.Hosts = []string{"one", "two", "three"}
```

Comment:
This had slipped my attention earlier. In 33ec84e I've reorged all test cases into the same block, segmented by comment blocks instead

Reply:
Looking good!